Cookbook/Technical-Documents/LabelsInPRs.md

```diff
@@ -0,0 +1,118 @@
+# How to add Labels to a Pull Request
+
+
+We use labels in our Pull Requests to be able to categorize and prioritize our work. Some are also used to trigger our CI jobs.
+
+
+## 1. The labels
+
+At the moment our labels can be separated into the following groups:
+
+- [Product verticals](#product-verticals)
+
+- [Project horizontals](#project-horizontals)
+
+- [Branding](#branding)
+
+- [Status](#status)
+
+- [Emergency](#emergency)
+
+- [Extra](#extra)
+
+
+### Product verticals
+A product vertical corresponds to a big feature of the product. 
+
+- <span style="background-color:#2BBBBB; color:black; font-weight:600">Health Management 🍄</span> - any work done in the health management vertical
```

Comment:
I just realized that those `background-color`/`color`/`font-weight` CSS styles don't render on GitHub, so despite all the effort you put in matching the label colors and making them look nice, it won't be visible in the document 😢 

Not sure if there's any official alternative though, nothing official in the GitHub-Flavored Markdown (GFM) documentation.

Only workaround that I've found is to use a placeholder image service, like this:

```md
 ![#2bbbbb](https://placehold.it/30x15/2bbbbb?text=+) `Health Management 🍄`
```
Which renders like this:

* ![#2bbbbb](https://placehold.it/30x15/2bbbbb?text=+) `Health Management 🍄`

Not ideal, but since GitHub markdown don't support background/foreground colors anyway...

Or alternatively we can just accept that those won't be nicely colored (but then maybe we should still wrap the label names between backticks to clearly isolate them from the rest of the text. A third alternative would be to convert those bullet points to a markdown table (column 1 = label name, column 2 = description)

Reply:
@AnnKatF Ohhh wait better solution to that limitation: Use shields.io of course! And that's a service hosted by GitHub and used in many README on GitHub to have badges already, so it fits better actually:

`https://img.shields.io/badge/-Health%20Managment%20%F0%9F%8D%84-2bbbbb` gives:

![](https://img.shields.io/badge/-Health%20Managment%20%F0%9F%8D%84-2bbbbb)

You can create your owns either my building the URL yourself, or use their dedicated form [here](https://shields.io/#your-badge) – keeping `label` empty but filling the `message` and `color` fields appropriately